File: Proyecto/UPCNETSystemWebService/src/java/com/upc/upcnet/entidades/Seccion.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.upc.upcnet.entidades;

/**
 *
 * @author ALEX
 */
public class Seccion {
    private String idSecclase;
    private String idSeccion;
    private String idAlumno;
    private String idCurso;
    private float pc1;
    private float pc2;
    private float ep;
    private float ef;
    private String retirado;
    private String inasistencias;
    private String delegado;

    public String getIdSecclase() {
        return idSecclase;
    }

    public void setIdSecclase(String idSecclase) {
        this.idSecclase = idSecclase;
    }

    public String getIdSeccion() {
        return idSeccion;
    }

    public void setIdSeccion(String idSeccion) {
        this.idSeccion = idSeccion;
    }

    public String getIdAlumno() {
        return idAlumno;
    }

    public void setIdAlumno(String idAlumno) {
        this.idAlumno = idAlumno;
    }

    public String getIdCurso() {
        return idCurso;
    }

    public void setIdCurso(String idCurso) {
        this.idCurso = idCurso;
    }

    public float getPc1() {
        return pc1;
    }

    public void setPc1(float pc1) {
        this.pc1 = pc1;
    }

    public float getPc2() {
        return pc2;
    }

    public void setPc2(float pc2) {
        this.pc2 = pc2;
    }

    public float getEp() {
        return ep;
    }

    public void setEp(float ep) {
        this.ep = ep;
    }

    public float getEf() {
        return ef;
    }

    public void setEf(float ef) {
        this.ef = ef;
    }

    public String getRetirado() {
        return retirado;
    }

    public void setRetirado(String retirado) {
        this.retirado = retirado;
    }

    public String getInasistencias() {
        return inasistencias;
    }

    public void setInasistencias(String inasistencias) {
        this.inasistencias = inasistencias;
    }

    public String getDelegado() {
        return delegado;
    }

    public void setDelegado(String delegado) {
        this.delegado = delegado;
    }
    
}

File: Proyecto/UPCNETSystemCliente/UPCNETSystemCliente/Business Entities/Seccion.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace UPCNETSystemCliente.Business_Entities
{
    public class Seccion
    {
        private String IdSecClase;
        private String IdSeccion;
        private String IdAlumno;
        private String IdCurso;
        private String Pc1;
        private String Pc2;
        private String Pc3;
        private String Ep;
        private String Ef;
        private String Retirado;
        private String Inasistencias;

        public Seccion(string idSecClase, string idSeccion, string idAlumno, string idCurso, string pc1, string pc2, string pc3, string ep, string ef, string retirado, string inasistencias)
        {
            IdSecClase = idSecClase;
            IdSeccion = idSeccion;
            IdAlumno = idAlumno;
            IdCurso = idCurso;
            Pc1 = pc1;
            Pc2 = pc2;
            Pc3 = pc3;
            Ep = ep;
            Ef = ef;
            Retirado = retirado;
            Inasistencias = inasistencias;
        }

        public String getIdSecClase() { return IdSecClase; }
        public String getIdSeccion() { return IdSeccion; }
        public String getIdAlumno() { return IdAlumno; }
        public String getIdCurso() { return IdCurso; }
        public String getPc1() { return Pc1; }
        public String getPc2() { return Pc2; }
        public String getEp() { return Ep; }
        public String getEf() { return Ef; }
        public String getRetirado() { return Retirado; }
        public String getInasistencias() { return Inasistencias; }
           

    }
}